src/node_server_actions_3.cpp

```diff
@@ -271,7 +271,7 @@ void node_server::execute_solver(bool scf, node_count_type ngrids) {
 			output_all(this, "final", output_cnt, true);
 		}
 		if (get_analytic() != nullptr) {
-      if (!(opts().stop_step > 1)) // Pure performance measurements - skip analytics 
+      if (!(opts().stop_step > 1) && !opts().disable_analytic) // Pure performance measurements - skip analytics 
```

Comment:
I'm a little bit confused regarding this part.
What is the meaning of having iterations if without hydro there is no evolution?
I also prefer to be consistent with the notation of opening and closing brackets even for one line.

Reply:
Supporting multiple iterations in the gravity-only scenario was added in #389 as I wanted a convenient way to average the runtime over multiple FMM iterations without having to restart Octo-Tiger each time (as I need to run about 50 iterations to get a good average). So this is merely here for performance measurements, hence it skips the analytics part as well. If you look at the file you'll see that Octo-Tiger stores the FMM iteration runtimes in that function and prints and average afterwards.

Reply:
ok, so maybe then we can remove the stop_step condition and leave only the more explicit disable analytic condition?

Reply:
I'd be fine with that! I only used the stop_step condition in #389 because there was no disable_analytic flag yet